### PR TITLE
[ISSUE #563] fix pr #606 cause ci fail

### PR DESCRIPTION
**Pull request by chuntaojun**
## Checklist

- [ ] Add copyright holder at the beginning of .class file if it is new.
- [ ] Add information of this PR to CHANGELOG.md in root of project.
- [ ] All junit tests passing.
- [ ] Coverage from `Codecov Report` should not decrease.

## Checklist (Optional)

- [ ] Will Pull Request to branch of 2020.0 and hoxton.
- [ ] Add documentation in javadoc in code or comment below the PR if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,4 +6,3 @@
 - [Optimize: remove useless code for rest template router](https://github.com/Tencent/spring-cloud-tencent/pull/601)
 - [Optimize: optimize configuration conditional & optimize config data tips](https://github.com/Tencent/spring-cloud-tencent/pull/603)
 - [Automatically transmit some headers specified by a environment varaible that directly defines header keys.](https://github.com/Tencent/spring-cloud-tencent/pull/606)
-- [Optimize: Maybe remove Chinese characters](https://github.com/Tencent/spring-cloud-tencent/pull/607)
```
Comment (SkyeBeFreeman) [inline CHANGELOG.md:9]: 添加你的pr信息，不要删除其他开发者的pr信息～

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]

**Review by SkyeBeFreeman — APPROVED** [review]

**Comment by pandaapo** [comment]
不好意思，没及时处理。感谢。

**Review by pandaapo — COMMENTED** [review]
做了如下一点补充

File: spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/metadata/StaticMetadataManager.java [hunk 2]
```diff
@@ -107,7 +107,7 @@ private void parseEnvMetadata() {
 			if (StringUtils.isNotBlank(key) && key.startsWith(ENV_METADATA_PREFIX)
 					&& !key.equals(ENV_METADATA_CONTENT_TRANSITIVE)) {
```
Comment (pandaapo) [inline spring-cloud-tencent-commons/src/main/java/com/tencent/cloud/common/metadata/StaticMetadataManager.java:108]: `ENV_METADATA_CONTENT_RAW_TRANSHEADERS`改成`ENV_TRAFFIC_CONTENT_RAW_TRANSHEADERS`后，这个判断要做点补充，不然会把`ENV_TRAFFIC_CONTENT_RAW_TRANSHEADERS`排除掉。

已在PR #618 中处理。

**Pull request merged**